### DISCUSSION TITLE
Drop `zeit/next.js` from `monorepo:nextjs` preset

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#16409] [Suggest an Idea]
https://github.com/renovatebot/renovate/blob/3589d0e222bd2b188a2336df68e0aacbd22f81e1/lib/config/presets/internal/monorepo.ts#L157-L160

It's been a while since Zeit renamed themselves to Vercel. Maybe we can drop the line that references the old repository?

Clicking on `https://github.com/zeit/next.js` correctly redirects to the "new" repository at: `https://github.com/vercel/next.js`. So we could also leave it as-is if we really wanted.

- [`zeit` organization profile on GitHub](https://github.com/zeit) shows the `zeit` org has no public repos anymore
- [Vercel blog about Zeit to Vercel rename](https://vercel.com/blog/zeit-is-now-vercel)

**Comment by rarkins** [comment]
It depends on whether any old releases still refer to that repo or not. If none that users are likely to be using, then we can drop
